src/viam/services/motion/client.py

```diff
@@ -171,6 +172,7 @@ async def move_on_map(
         destination: Pose,
         slam_service_name: ResourceName,
         configuration: Optional[MotionConfiguration] = None,
+        obstacles: Optional[Iterable[Geometry]] = [],
```

Comment:
Actually that's a good catch, it should be `None`

Reply:
Also, is there a way to double check that this type should indeed be `Optional[Iterable[T]]` and not `Iterable[T]` or `Sequence[T]`?

Reply:
Are `obstacles` required? Because if not, keeping it as `Optional[...]` would be better as it more explicitly uses the type to suggest that it is indeed *optional*

Reply:
they are not required! :D